db/seeds.rb

```diff
@@ -47,12 +47,12 @@
 
 if Region.all.empty?
   puts 'Loading regions'
-  Rake::Task['shf:load_regions'].invoke
+  Rake::Task['shf:load_regions'].execute
```

Comment:
I figured you probably knew about the differences with dependencies, but included the example just in case you or someone else reading this didn't.   I knew about the difference with the dependencies, but didn't really know the details about the differences with the compiling and running more than once.   So thanks for explaining that.  Now I understand `reenable` more deeply.

The dependencies isn't a big deal with this one, but I'm glad we've got it working cleanly as possible now. 

The power of reviews:  We help each other level up. :-)